### DISCUSSION TITLE
Docker Pretix Update 4.3.0

### DISCUSSION THREAD

**Discussion by Shakesbeer121** [pretix/pretix#2233] [Q&A]
Hello,

I have a small scale docker installation of Pretix running on my server. When I now try to update it to 4.3.0 using the simple three steps from your documentation (https://pretix.readthedocs.io/en/latest/admin/installation/docker_smallscale.html), I get the following message:

![unnamed (1)](https://user-images.githubusercontent.com/16540336/135637363-c954c84c-7b2c-4ae3-b3df-d819a40b85e4.png)

Is there a problem with your docs or is it me? It would be very nice if you could give me advice on how to fix it. Thank you for your time.

Yours sincerely,
Ludwig

**Comment by raphaelm** [comment]
This can safely be ignored!
